source/agora/consensus/Validation.d

```diff
@@ -993,10 +994,14 @@ unittest
     auto gen_key = getGenesisKeyPair();
     assert(GenesisBlock.isValid(GenesisBlock.header.height, Hash.init, null));
     auto gen_hash = GenesisBlock.header.hashFull();
+    auto enroll_man = new EnrollmentManager(":memory:", gen_key);
+    scope (exit) enroll_man.shutdown();
+    Enrollment[] enrollments;
```

Comment:
Most of the changes in this file seems to have no effect. No `Enrollment` is every registered, thus all calls to `getUnregisteredEnrollments` will return `null`.

Reply:
I used `testMakeNewBlock`